docs/pages/versions/unversioned/config/metro.mdx

```diff
@@ -4,15 +4,40 @@ title: metro.config.js
 
 See more information about **metro.config.js** in the [customizing Metro guide](/guides/customizing-metro/).
 
+## Envrionment Variables
+
+> Since SDK 49
+
+Environment variables can be loaded using `.env` files. Files are loaded according to the [standard `.env` file resolution](https://github.com/bkeepers/dotenv/blob/c6e583a/README.md#what-other-env-files-can-i-use).
```

Comment:
```suggestion
Environment variables can be loaded using **.env** files. These files are loaded according to the [standard **.env** file resolution](https://github.com/bkeepers/dotenv/blob/c6e583a/README.md#what-other-env-files-can-i-use).
```

[Filenames as bold text as per the writing styelguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)